src/Layout/Presentation/RemainderQuotients/RemainderQuotients.tsx

```diff
@@ -1,7 +1,8 @@
 import * as React from "react";
 import ReactTable, { Column } from "react-table";
-import { LevelingSeat, DistrictResult, DistrictQuotients } from "../../../computation";
+import { LevelingSeat, DistrictResult, DistrictQuotients, AlgorithmType } from "../../../computation";
 import { norwegian } from "../../../utilities/rt";
+import { isQuotientAlgorithm } from "../../../computation/logic";
```

Comment:
I feel like this is indicative of a problem with our architecture. It's strange that logic is computed and embedded here rather than the output simply being displayed.

Reply:
It is a result of us calculating the closest seat on the display side, rather than in the logic.

Reply:
Why do we do that? Do we have to, or is it potentially fixable in the future?

Reply:
Good question, I did not write that code. I believe you did.